Description: Rename libquazip library name for libquazip transition
 For libquazip to transition from the Qt4 to the Qt5 version as default,
 the linker library name needs to be adjusted in the cen64-qt sources.
 This patch modifies cen64-qt.pro to allow cen64-qt to work with both
 the old and the new library name so the package can still be backported
 to older Debian releases prior to the libquazip transition after
 adjusting build dependencies in debian/control.
 .

--- cen64-qt-20150915-alpha.orig/cen64-qt.pro
+++ cen64-qt-20150915-alpha/cen64-qt.pro
@@ -63,7 +63,11 @@ win32|macx|linux_quazip_static {
     } else {
         # Debian distributions use a different library name for Qt5 quazip
         system("uname -a | grep -E 'Debian|Ubuntu' > /dev/null") {
-            LIBS += -lquazip-qt5
+              system("dpkg -l | grep libquazip5-dev | grep ^ii > /dev/null") {
+                  LIBS += -lquazip5
+              } else {
+                  LIBS += -lquazip-qt5
+              }
         } else {
             LIBS += -lquazip5
         }
